src/components/Donation/constants/placeholders.ts

```diff
@@ -0,0 +1 @@
+export const EMPTY_SPACE = ' ';
```

Comment:
I would said this is a bit overkill...
What are the benefits of doing this from your point of view? I see the usage is 

```
 errors?.join(EMPTY_SPACE)
```

I will rather do 

```
 errors?.join(' ')
```

_But maybe I am missing something..._